src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultTagHelperResolver.cs

```diff
@@ -19,46 +21,82 @@ public DefaultTagHelperResolver(bool designTime)
         public override IReadOnlyList<TagHelperDescriptor> GetTagHelpers(Compilation compilation)
         {
             var results = new List<TagHelperDescriptor>();
+            var errors = new ErrorSink();
```

Comment:
Yep. We need to do a refactor on `THD` or least evaluate the design. Let's plan on doing this today.